data/human/hails.txt

```diff
@@ -2794,6 +2797,454 @@ phrase "friendly pirate"
+		"We couldn't even count that many ships"
+		"You shoulda been there"
+		"You shoulda heard them scream"
+		"You shoulda brought along your guns"
+		"I'm not kidding"
+		"I'm not joking"
+		"I'm not one to joke"
+		"It's not a joke"
+		"The Man sure pays his dues"
+		"We smoked their defense fleet"
+		"We obliterated their defense fleet"
+		"We decimated their defense fleet"
+		"We destroyed their defense fleet"
+		"They weren't even a match for us"
+		"We didn't lose a single ship"
+		"We beat them to a pulp"
+		"We sent them packing"
+		"It was too easy"
+		"That was one hell of a fight"
+		"That battle's going into the history books"
+		"I'm sure I blew up more than a dozen ships"
+		"Now that's who you're facing"
+		"That's who you're facing"
+	word
+		"!"
+
+phrase	"pirate big raid 2"
+	word
+		"the "
+	word
+		"fattest"
+		"juiciest"
+		"plumpest"
+		"greatest"
+		"hugest"
+		"gnarliest"
+		"biggest"
+		"hugest"
+		"craziest"
+		"juiciest"
+		"most cosmic"
+		"most enormous"
+		"most epic"
+		"most fat"
+		"most juicy"
+		"most greatest"
+		"most biggest"
+		"most fattest"
+		"most nasty"
+		"most gnarly"
+		"most plump"
+		"most ambitious"
+		"most crazy"
+	word
+		" "
+	word
+		"raid"
+		"assault"
+		"sack"
+		"plundering"
+		"looting"
+		"storming"
+		"sacking"
+	word
+		" of "
+		" on "
+		" at "
+	phrase
+		"north pirate target"
+		"south pirate target"
+		"core pirate target"
+	word
+		""
+		" ever"
+		" in their history"
+		" in history"
+		" of all time"
+		". We made off with so many ships"
+		". We made off with so much tribute"
+		". We made off with such a big haul"
+		". We couldn't fit everything onto our ships"
+		". We're richer than David Turner"
+		". We made off with so many credits"
+		". We're easily the richest pirates this side of Earth"
+		". We're easily the richest pirates this side of Hephaesteus"
+		". We've got more ships than we can count"
+		". We've got more credits than we can count"
+		". We've got more guns than we can count"
+		". You shoulda been there"
+		". You shoulda heard them scream"
+		". You shoulda brought along your guns"
+		". The Man sure pays his dues"
+		". We smoked them"
+		". We obliterated them"
+		". We decimated them"
+		". We destroyed them"
+		". They weren't even a match for us"
+		". We sent them packing"
+		". We didn't lose a single ship"
+		". It was too easy"
+		". I'm sure I blew up more than a dozen ships"
+		". That was one hell of a fight"
+		". I'm not kidding"
+		". I'm not joking"
+		". I'm not one to joke"
+		". It's not a joke"
+	word
+		"!"
+
+phrase "north pirate target"
+	word
+		"Farpoint"
+		"Shroud"
+		"Sunracer"
+		"Prime"
+		"Follower"
+		"Freedom"
+		"Haze"
+		"Memory"
+		"Stormhold"
+		"Haven"
+		"Zenith"
+# A few pirates make it up as far north as Asgard, and they have ships like Corvettes and Aeries which they must get from the Deep by raiding or capture. The pirate planets are there because of pirate band infighting. 
```

Comment:
Comments should be above the thing they're commenting on.
```suggestion
# A few pirates make it up as far north as Asgard, and they have ships like Corvettes and Aeries which they must get from the Deep by raiding or capture. The pirate planets are there because of pirate band infighting. 
phrase "north pirate target"
	word
		"Farpoint"
		"Shroud"
		"Sunracer"
		"Prime"
		"Follower"
		"Freedom"
		"Haze"
		"Memory"
		"Stormhold"
		"Haven"
		"Zenith"
```